src/Illuminate/Queue/Console/WorkCommand.php

```diff
@@ -91,6 +91,17 @@ public function fire()
      */
     protected function runWorker($connection, $queue, $delay, $memory, $daemon = false)
     {
+        $timeout = $this->option('timeout');
```

Comment:
Maybe `extension_loaded('pcntl')` is enough and these are redundant?
`function_exists('pcntl_alarm') && function_exists('pcntl_signal')`